### DISCUSSION TITLE
after windows update(10 to 11) my plots apear missing plots

### DISCUSSION THREAD

**Discussion by tonmarioh** [Chia-Network/chia-blockchain#18040] [Support]
after windows update(10 to 11) my plots apear missing plots

**Comment by BrandtH22** [comment]
Hey @tonmarioh, as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

I highly recommend uninstalling and reinstalling chia now that the device is upgraded. After launching chia you might need to readd the plot directories but so long as the plots are visible in the file explorer on windows 11 the chia client will be able to read them.

Let us know what questions arise and make it an excellent day,
Brandt H

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
